src/app/components/common/topbar/topbar.component.ts

```diff
@@ -313,7 +322,9 @@ export class TopbarComponent implements OnInit, OnDestroy {
   }
 
   onGoToLegacy() {
-    this.dialogService.confirm(T("Switch to Legacy User Interface?"), T(""), true, T("Continue")).subscribe((res) => {
+    this.dialogService.confirm(T("Warning:"), 
+      T("The Legacy User Interface is deprecated. The system should be managed from the new UI."), 
+      true, T("I understand")).subscribe((res) => {
```

Comment:
Not clear what the "I understand" does--enable the legacy UI, or just make the warning go away?

Reply:
Changed to 'Continue to Legacy UI' on the button (hope UI is ok here, since the message above calls it the user interface.